codegen-on-oss/codegen_on_oss/snapshot/context_snapshot.py

```
@@ -0,0 +1,205 @@
+            temp_path = Path(temp_dir)
+            zip_path = temp_path / f"{snapshot_id}.zip"
+            
+            # Download the snapshot
+            s3_client = bucket_store.s3_client
+            
+            # List objects to find the snapshot
+            response = s3_client.list_objects_v2(
+                Bucket=bucket_name,
+                Prefix=f"{bucket_store.key_prefix}/snapshots",
+            )
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)